### PR TITLE
SISRP-15495, concurrent enrollment students see non-student message on profile page

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.berkeley.edu/browse/SISRP-15495

PR needs @raydavis or similar sign-off. Is wording okay? Is my assumption correct that concurrent students can use CalNet directory to update personal info?

### CODE REVIEW

**Comment by christianvuerings** [comment]
Lgtm but waiting for @raydavis

**Comment by raydavis** [comment]
We have to either wait for confirmation that concurrent-enrollment students _do_ have the ability to edit their CalNet directory entries, or revise the text.

**Comment by johncrossman** [comment]
PR updated; boilerplate simply does not show if `api.user.profile.roles.concurrentEnrollmentStudent` See latest comments in Jira.

File: src/assets/templates/widgets/profile/basic_nonstudent.html [hunk 1]
```diff
@@ -3,11 +3,11 @@
   data-ng-controller="BasicController"
   data-cc-spinner-directive="basicInformation.isLoading">
 
-  <div class="cc-page-widget-profile-note">
+  <div class="cc-page-widget-profile-note" data-ng-if="!api.user.profile.roles.concurrentEnrollmentStudent">
     <i class="fa fa-exclamation-triangle cc-icon cc-icon-gold" aria-hidden="true"></i>
     <strong>Note:</strong> Although active students will use CalCentral to view and update personal data in
     UC Berkeley's Student Information System (SIS), non-students should continue to rely on the
-    <a href="https://calnet.calnet.berkeley.edu/directory/update/">CalNet Directory</a>.
+    <a href="https://calnet.berkeley.edu/directory/update/">CalNet Directory</a>.
```
Comment (raydavis) [inline src/assets/templates/widgets/profile/basic_nonstudent.html]: Thanks for fixing that URL!

**Pull request merged**